service/server.py

```diff
@@ -18,20 +28,72 @@ def healthcheck():
     return "OK"
 
 
+def get_property_address(postcode):
+    client = Elasticsearch([ELASTIC_SEARCH_ENDPOINT])
+    search = Search(
+        using=client, index='landregistry', doc_type='property_by_postcode')
+    query = search.filter('term', postcode=postcode)
+    return query.execute().hits
```

Comment:
This is fine, provided we only allow full-length postcodes (otherwise we would have too many results).